src/providers/lambda.ts

```diff
@@ -221,13 +233,13 @@ export class LambdaRunner extends Construct implements IRunnerProvider {
   getStepFunctionTask(parameters: RunnerRuntimeParameters): stepfunctions.IChainable {
     return new stepfunctions_tasks.LambdaInvoke(
       this,
-      this.label,
+      this.labels.join(', '),
       {
         lambdaFunction: this.function,
         payload: stepfunctions.TaskInput.fromObject({
           token: parameters.runnerTokenPath,
           runnerName: parameters.runnerNamePath,
-          label: this.label,
+          label: this.labels.join(','),
```

Comment:
For consistency, I would rename this too:
```suggestion
          labels: this.labels.join(','),
```